apps/files/js/filelist.js

```diff
@@ -3859,7 +3859,7 @@
 				this._newFileMenu = new OCA.Files.NewFileMenu({
 					fileList: this
 				});
-				$('.actions').append(this._newFileMenu.$el);
+				this.$el.find('#controls .actions').append(this._newFileMenu.$el);
```

Comment:
Did you mean:

```suggestion
				this.$el.find('.files-controls .actions').append(this._newFileMenu.$el);
```

?

Reply:
Oh yes my bad, I've made that PR on an older version and I didn't notice the change, I'll fix that